packages/user-interface/src/tests/mock/data/CasesPageRequests.mock.tsx

```diff
@@ -1,9 +1,11 @@
-import { FetchResult, GraphQLRequest } from "@apollo/client";
-import { QUERY_GET_ZAKEN } from "@nl-portal/nl-portal-api";
+import { GraphQLRequest } from "@apollo/client";
+import { GetZakenQuery, QUERY_GET_ZAKEN } from "@nl-portal/nl-portal-api";
+
+type GetZakenResult = { data: GetZakenQuery };
 
 const getZaken: {
   request: GraphQLRequest;
-  result: FetchResult<Record<string, any>>;
+  result: GetZakenResult;
```

Comment:
Hier kan ook gewoon dat object in toch? gebruikt hem maar 1x

Reply:
Klopt, heb ik aangepast.